shotover-proxy/src/protocols/cassandra_protocol2.rs

```diff
@@ -416,7 +420,7 @@ impl CassandraCodec2 {
                                 with_hints: _,
                             } = &s.from.get(0).unwrap().relation
                             {
-                                namespace = name.0.clone();
+                                namespace = name.0.clone().iter().map(|a| a.to_string()).collect();
```

Comment:
remove the `clone()`

Reply:
Wait uhhh... before we change these clones, does to_string do what we think it does?

Reply:
It does not: https://doc.rust-lang.org/std/string/trait.ToString.html
It performs display

Reply:
Look at idents Display impl to see why this would have been bad.
![image](https://user-images.githubusercontent.com/5120858/140004801-6736a4da-730a-4695-80cc-e137ae628749.png)

Reply:
So what we want to do for all of these cases is `namespace.0.iter().map(|a| a.value.clone()).collect()`